src/jmespath.net.parser/jmespath.net.parser.csproj

```diff
@@ -1,5 +1,4 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-

```

Comment:
Nitpick: please let’s keep a blank line after<Project>.
No real reason, but I guess it also avoids extra noise in this commit.

Reply:
Corrected in latest checkin